src/github.com/getlantern/flashlight/client/config.go

```diff
@@ -20,6 +21,10 @@ type ClientConfig struct {
 	ChainedServers map[string]*ChainedServerInfo
 	Peers          map[string]*nattywad.ServerPeer // keyed to peer id (e.g. XMPP JID)
 	MasqueradeSets map[string][]*fronted.Masquerade
+
+	OpenUi    bool              // Flag specifying whether or not the UI should be automatically open in the browser
+	UiAddr    string            // UI HTTP server address
```

Comment:
I just put it there for now since we're only using it in get-mode, but out of the client and into the parent config.